corellium/adapter/src/test/kotlin/flank/corellium/adapter/ExecuteAndroidTestPlanKtTest.kt

```diff
@@ -0,0 +1,65 @@
+package flank.corellium.adapter
+
+import flank.corellium.api.AndroidTestPlan
+import flank.corellium.client.console.Console
+import flank.corellium.client.console.close
+import flank.corellium.client.console.flowLogs
+import flank.corellium.client.console.sendCommand
+import flank.corellium.client.core.Corellium
+import flank.corellium.client.core.connectConsole
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.channelFlow
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert
+import org.junit.Test
+
+class ExecuteAndroidTestPlanKtTest {
```

Comment:
Please add `unmockkAll()` tear down method -- unmocked tests may interact between test suites (we had this issue couple of times in the past)

Reply:
Fixed